src/Avatar.php

```diff
@@ -66,7 +66,7 @@ public function filter_avatar( $avatar, $id_or_email, $size, $default, $alt, arr
 			return $avatar;
 		}
 
-		if ( ! ( empty( $args['found_avatar'] ) || $args['force_default'] ) ) {
+		if ( ! ( $args['found_avatar'] || $args['force_default'] ) ) {
```

Comment:
I'm new to wordpress development, not sure why need to check if the flag is empty?